docs/HTML.md

```diff
@@ -0,0 +1,93 @@
+### 1. `<header>` -- Header 
+* *type:* block-level
+* *content:* Flow content/palpable content
+* *support:* all browsers
+* *example:* 
+```
+ <header> Logo </header>
+```
+<header> Logo </header>
+
+### 2. `<hr>` -- Horizontal Rule
+* *type:* block-level
+* *content:* Flow Content
+* *support:* all browsers
+* *example:*
+``` <p> Some text </p>
+ <hr>
+<p> some other text </p> ```
+<p> some Text </p>
+<hr>
+<p> some other text </p>
+
```

Comment:
Oh, and another clue is how the syntax highlighting doesn't change colors past this point.